src/robusta/integrations/slack/sender.py

```diff
@@ -426,7 +433,7 @@ def __create_finding_header(
             title = f"<{finding.get_investigate_uri(self.account_id, self.cluster_name)}|*{title}*>"
         return MarkdownBlock(
             f"""{status_name} {sev.to_emoji()} *{sev.name.capitalize()}*
-{title}"""
+{title}{mention}"""
```

Comment:
Can you please add test for this use case? Thanks

Reply:
fixed